static/main/main.js

```diff
@@ -7,12 +7,9 @@ process.env.PATH += ':/usr/local/bin';
 var mainWindow;
 
 app.on('ready', function () {
-
-    var atomScreen = require('screen');
-    var size = atomScreen.getPrimaryDisplay().workAreaSize;
     mainWindow = new BrowserWindow({
-        width: size.width,
-        height: size.height,
+        width: 700,
+        height: 450,
```

Comment:
Why we can't use display's size? Or replace it with `height: size.height || 700`?